### ISSUE TITLE
The hidden-until-hovered ui elements make wagtail admin almost unuseable on mouseless, touch based devices

### ISSUE THREAD

**Issue by eveningtree** [wagtail/wagtail#9782]
Context: We write on the website primarily through mobile devices like ipads, phones.

The problem specifically stings in these areas:
1. In the nav bar, when moving up the hierarchy. The full lineage doesn't show itself until tapped.
2. Performing actions on child pages: The edit, view live, more buttons are hidden.

**Comment by eveningtree** [comment]
Is this because of some conscious decision on the UX?
If you need a detailed spec of the issue, with list of all places where the same problem comes up, I can type it up and send it over the weekend.

**Comment by lb-** [comment]
@eveningtree  thanks for flagging this issue - we have an ongoing discussion thread here https://github.com/wagtail/wagtail/discussions/9553

It might be good to keep this issue open for now but add some more specific (even with recordings) details of where you are finding this problem on the actual discussion. 

I recall seeing some css in place to ensure that 'visible on hover' items would only impact desktop devices and be always visible on mobiles. However, maybe that has not been done consistently.

Nonetheless, this feedback is really helpful, I cannot give any kind of timeline or direction on where this will head but the more detail and granular feedback from you and others in the community the better.

**Comment by thibaudcolas** [comment]
Hi @eveningtree, thank you for reporting this. Please do report any specific occurrences of this issue if you run into it. There are a lot of components in Wagtail following this pattern, however they’re all meant to be using the `@media (hover: hover)` media query so we only hide the elements for devices supporting hover interactions.

**Comment by FlipperPA** [comment]
@thibaudcolas While I wouldn't say this makes it unusable by any stretch, I have heard reports from our users who spend a fair amount of their week in the Wagtail admin that this slows them down noticeably. It removes a contextual clue that used to be just a glance that lets the user know where they are, and requires extra mouse movement and a click to use the breadcrumbs. For users that have lots of tabs open, having the breadcrumbs open when navigating between tabs is extremely useful. It'd be great if we could either revert to the previous behavior for breadcrumbs in particular or give the user an option to keep them expanded across the admin.

**Comment by thibaudcolas** [comment]
Thanks for the nudge @FlipperPA, can confirm we are revisiting breadcrumbs and the best place to feed back on this in #9498. 

@eveningtree for anything else, we could definitely use your longer feedback list, as we have very few users reporting they use Wagtail primarily on touch devices.

**Comment by spapas** [comment]
Hello friends, beyond the mouse usability problems, this is problematic with non-technical users: 

Yesterday i talked with one of my really-non-technical editors and wanted to teach her to "move" a page. She really couldn't because the "move" option is displayed only when you move the mouse cursor over the page you wanna move. this is cool for aesthetic reasons but problematic for people that don't understand "hovering". 

I think it would be useful either to make these always visible or add some kind of simple way to make them visible when needed. 
See for example image below:

![image](https://github.com/wagtail/wagtail/assets/3911074/c6b96fdf-9292-4d89-bf3d-81ebcc36d8ce)

**Comment by Stormheg** [comment]
For anyone running into this and want a quick-and-dirty solution: you can override Wagtail's admin css to make the actions always visible.

Add this to your projects `wagtail_hooks.py`

```python
from wagtail import hooks

@hooks.register('insert_global_admin_css')
def global_admin_css():
    return "<style>.listing .actions { visibility: visible !important; }</style>"
```

This makes the actions always visible in the page explorer, snippets listing and modeladmin listing. Tested with Wagtail 5.0.

Be aware that overriding Wagtail in this way is not supported and this code may break in future versions of Wagtail. You have been warned.
